ui/app/components/job-status/individual-allocation.hbs

```diff
@@ -19,6 +19,8 @@
         <span class="alloc-health-indicator">
           {{#if (eq @health "healthy")}}
             <FlightIcon @name="check" @color="white" />
+          {{else if (eq @health "unhealthy")}}
+            <FlightIcon @name="x" @color="white" />
```

Comment:
This is the indicator for when an allocation is un-aggregated/summarized, in say a small enough cluster that each allocation gets its own square.